### PR TITLE
rework on navbar items

### PR DESCRIPTION
**Pull request by dhmlau**
### Navbar items
- Remove "GraphQL" and "Test Your APIs"
- Add the icons for GitHub, Twitter and Slack

Screen Shot 2020-04-17 at 3 28 26 PM

### resources.html
- Add back the reference for GraphQL and Test Your APIs

Screen Shot 2020-04-17 at 3 34 21 PM

### CODE REVIEW

**Comment by dhmlau** [comment]
cc @achrinza

**Review by raymondfeng — APPROVED** [review]

File: resources.html [hunk 1]
```diff
@@ -159,6 +160,51 @@ <h5 class="card-title">Tutorial for Application Developers</h5>
...
+
+            <div class="row text-center" style="justify-content: center">
+                <div class="col-sm-12 col-md-6">
+                    <a class="card-2" href="openapi-to-graphql.html" target="_blank" rel="noopener noreferrer">
+                        <div class="row">
+                            <div class="col-sm-3">
+                                <img class="icon" src="images/homepage/conveyorbelt-2x.png" alt="OpenAPI-to-GraphQL Icon"/>
+                            </div>
+                            <div class="col-sm-9">
+                                <h5 class="card-title">openapi-to-graphql</h5>
```
Comment (nabdelgadir) [inline resources.html]: very small nitpick but can we keep the capitalization the same as the page's: `OpenAPI-to-GraphQL`

**Review by nabdelgadir — APPROVED** [review]

File: resources.html (hunk 1 above)
Comment (dhmlau) [reply resources.html]: I was thinking to keep the module name. But can change it to `OpenAPI-to-GraphQL`.

**Review by jannyHou — APPROVED** [review]
👏 LGTM

**Pull request merged**